### PR TITLE
#43 import yoast content

### PR DESCRIPTION
**Pull request by nickmoreton**

### CODE REVIEW

File: wagtail_wordpress_import/test/tests/test_wordpress_item.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 import json
-from django.test import TestCase
+from django.test import TestCase, override_settings
```
Comment (kaedroho) [inline wagtail_wordpress_import/test/tests/test_wordpress_item.py]: Looks like this import didn't get used

File: wagtail_wordpress_import/importers/wordpress.py [hunk 2]
```diff
@@ -230,16 +220,16 @@ def prefilter_content(self, content):
         FILTERS ARE CUMULATIVE
         cache the result of each filter which is run on the output from the previous filter
         """
-        filter_config = getattr(
-            settings, "WAGTAIL_WORDPRESS_IMPORT_PREFILTERS", DEFAULT_PREFILTERS
-        )
+        # filter_config = getattr(
```
Comment (kaedroho) [inline wagtail_wordpress_import/importers/wordpress.py]: Looks like we can delete this

File: docs/yoast.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+## Enable the yoast plugin
+
+*Enabling the yoast plugin compatibility will override the default behaviour. The Wordpress XML file `<item><description>...` will be ignored*
+
+The yoast plugin feature can be enabled by adding the following configuration to your own settings
+
+```python
+WAGTAIL_WORDPRESS_IMPORT_YOAST_PLUGIN_ENABLED = True
+
+```
```
Comment (kaedroho) [inline docs/yoast.md:24]: Minor formatting issue. The ``` should be on the line above

File: docs/yoast.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+## Change the imported fields
+
+The package has a default mapping for Wordpress search descriptions. There is no need to add this configuration to your settings. If you do need to change the field names because your Wordpress site is setup differently, include the configuration below in your settings and adjust the right side of each parameter.
```
Comment (kaedroho) [inline docs/yoast.md]: I think it would be worth mentioning that the example below is the default value

**Review by kaedroho — COMMENTED** [review]
This looks good to me! Just some minor comments

**Comment by nickmoreton** [comment]
Hi Karl, I have updated the code for you comments.

**Review by kaedroho — APPROVED** [review]

**Pull request merged**